components/net/http_cache.rs

```diff
@@ -230,12 +230,15 @@ fn get_response_expiry(response: &Response) -> Duration {
         let max_heuristic = Duration::hours(24) - age;
         let heuristic_freshness = if let Some(last_modified) =
             // If the response has a Last-Modified header field,
-            // caches are encouraged to use a heuristic expiration value
-            // that is no more than some fraction of the interval since that time.
-            response.headers.typed_get::<LastModified>() {
+        // caches are encouraged to use a heuristic expiration value
+        // that is no more than some fraction of the interval since that time.
+            response.headers.typed_get::<LastModified>()
+        {
```

Comment:
Hmm, this format looks wrong to me 🤔 Should I file an issue to rustfmt?

Reply:
[This playground](https://play.rust-lang.org/?version=nightly&mode=debug&edition=2018&code=fn%20main()%20%7B%0A%20%20%20%20let%20o_num%20%3D%20Some(123)%3B%0A%20%20%20%20%0A%20%20%20%20let%20x%20%3D%20if%20let%20Some(n)%20%3D%0A%20%20%20%20%20%20%20%20%2F%2F%20this%20is%20a%20test%20comment%0A%20%20%20%20%20%20%20%20%2F%2F%20to%20see%20if%20rustfmt%20will%20break%0A%20%20%20%20%20%20%20%20%2F%2F%20after%20using%20the%20lateset%20version%0A%20%20%20%20%20%20%20%20o_num%20%7B%0A%20%20%20%20%20%20%20%20%20%20%20%20n%20*%202%0A%20%20%20%20%20%20%20%20%7D%20else%20%7B%0A%20%20%20%20%20%20%20%20%20%20%20%200%0A%20%20%20%20%20%20%20%20%7D%3B%0A%20%20%20%20%0A%20%20%20%20println!(%22Number%3A%20%7B%7D%22%2C%20x)%3B%0A%7D) can reproduce this format 👀